### PR TITLE
Added missing description for shipping types

### PR DESCRIPTION
**Pull request by Smit-Parmar**
I want to merge this change because it will add missing description for shipping.
Part of #13142 

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are either absent or guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migrations are either absent or optimized for zero downtime
* [ ] The changes are covered by test cases

### CODE REVIEW

File: saleor/graphql/shipping/types.py [hunk 1]
```diff
@@ -56,11 +56,17 @@
...
+        required=True, description="The ID of shipping method channel listing."
+    )
+    channel = graphene.Field(
+        Channel,
+        required=True,
+        description="The channel associated with the shipping method channel listing.",
+    )
+    maximum_order_price = graphene.Field(Money, description="Maximum order price.")
+    minimum_order_price = graphene.Field(Money, description="Minimum order price.")
+    price = graphene.Field(Money, description="Order price.")
```
Comment (korycins) [inline saleor/graphql/shipping/types.py:69]: It is a shipping price I guess

File: saleor/graphql/shipping/types.py [hunk 2]
```diff
@@ -252,9 +258,11 @@ def resolve_tax_class(root: ChannelContext[models.ShippingMethod], info):
 
 
 class ShippingZone(ChannelContextTypeWithMetadata[models.ShippingZone]):
-    id = graphene.GlobalID(required=True)
-    name = graphene.String(required=True)
-    default = graphene.Boolean(required=True)
+    id = graphene.GlobalID(required=True, description="The ID of shipping zone")
```
Comment (tomaszszymanski129) [inline saleor/graphql/shipping/types.py:261]:
```suggestion
    id = graphene.GlobalID(required=True, description="The ID of shipping zone.")
```

File: saleor/graphql/shipping/types.py [hunk 3]
```diff
@@ -252,9 +258,11 @@ def resolve_tax_class(root: ChannelContext[models.ShippingMethod], info):
...
+    name = graphene.String(required=True, description="Shipping zone name.")
+    default = graphene.Boolean(
+        required=True, description="The shipping zone by default."
```
Comment (tomaszszymanski129) [inline saleor/graphql/shipping/types.py:264]: Maybe something more like
```suggestion
        required=True, description="Indicates if the shipping zone is default one."
```
?

File: saleor/graphql/shipping/types.py (hunk 1 above)
Comment (Smit-Parmar) [reply saleor/graphql/shipping/types.py:69]: @maarcingebala can you confirm this please?

Comment (maarcingebala) [reply saleor/graphql/shipping/types.py:69]: Yes, it's the price of this shipping method in the associated channel.

**Comment by IKarbowiak** [comment]
Hi @Smit-Parmar, seems like you have some other changes on your PR too. Try to merge or rebase your branch with main.

File: saleor/graphql/shipping/types.py [hunk 4]
```diff
@@ -56,11 +56,17 @@
...
+        required=True, description="The ID of shipping method channel listing."
+    )
+    channel = graphene.Field(
+        Channel,
+        required=True,
+        description="The channel associated with the shipping method channel listing.",
+    )
+    maximum_order_price = graphene.Field(Money, description="Maximum order price.")
+    minimum_order_price = graphene.Field(Money, description="Minimum order price.")
+    price = graphene.Field(Money, description="Shipping price.")
```
Comment (maarcingebala) [inline saleor/graphql/shipping/types.py:69]: Maybe let's make it more descriptive - "Price of the shipping method in the associated channel".

**Review by korycins — APPROVED** [review]

**Pull request merged**